docs/user-guide/kubeconfig-file.md

```diff
@@ -94,151 +111,170 @@ users:
+ 1. Determine what context to use based on the following priorities:
+   1. Command line argument (kubectl config): Use the value specified for `context`.
+   1. Kubeconfig file: Use the value specified for `current-context`.
+   1. Set no context if the values are undefined.
+ 1. Determine what user and cluster information to use based on the following priorities:
+    Note: At this point, the context might be undefined. Also, this check runs twice: once for user and then again for cluster.
+   1. Command line argument (kubectl config): Use the value specified for `user` and `cluster`.
+   1. Kubeconfig file: If `context` is specified, then use the value in the nested `cluster`.
+   1. Set no user and cluster if the values are undefined.
+ 1. Determine the details of the cluster based on the following priorities:
+    Note: At this point, the cluster might be undefined. Also, only the values defined in the first found cluster are used and all subsequent values are ignored.
+   1. Command line argument (kubectl config): Use the values specified for `server`, `api-version`, `certificate-authority`, and `insecure-skip-tls-verify`.
+   1. Kubeconfig file: Use the values specified in `cluster`.
+   1. Ensure that a value for `server` is defined, otherwise throw an error.
+ 1. Determine the details of the user based on the following priorities:
+    Note: Only the values defined in the first found user are used and all subsequent values are ignored.
+   1. Command line argument (kubectl config): Use the values specified for `client-certificate`, `client-key`, `username`, `password`, and `token`.
+   1. Kubeconfig file: Use the values specified in `user`.
+   1. Ensure only a single authentication method is defined, otherwise throw an error.
+ 1. For any required information that's missing, either use default values or prompt the user for authentication information.
+
+## Managing kubeconfig files with `kubectl config`
+
+Use the `kubectl config` *`subcommand`* commands to easily create, add, update, or remove details from your kubeconfig files. See the [kubectl_config](kubectl/kubectl_config.md) reference topic for details about all the commands.
+
+### Creating kubeconfig files
+
+You can run `kube-up.sh` to create kubeconfig files or you can manually create them.
+
+If you create your cluster by running `kube-up.sh`, a kubeconfig file is created for you. See [Creating a Kubernetes Cluster](../getting-started-guides/README.md) for information about getting started with `kube-up.sh`.
+
+To manually create a kubeconfig file, you can either create the file itself or you can use the `kubectl config` commands. For example, you can use the example kubeconfig file in the section above as a template to create your own kubeconfig file. If you manually create the file yourself, you must name it `config` and then save it to your `$HOME/.kube/` directory.
```

Comment:
@jlowdermilk, i kept this in here but im wondering if we should just simplify and remove the content about manually creating the file yourself. Is that something we should continue to mention, do we recommend that method? Perhaps we can simplify this "creating a kubeconfig file" section down to only two options: either use kube-up or otherwise use kubectl config?